force-app/main/default/classes/CountTasksTest.cls

```diff
@@ -0,0 +1,206 @@
+/**
+ * Apex Test Class for all scenarios
+ * (insert, update, delete, close tasks,
+ * re open tasks, reparenting tasks)
+ */
+@isTest
+public class CountTasksTest {
+  @TestSetup
+  private static void makeData() {
+    List<Account> accounts = new List<Account>();
+
+    for (Integer i = 0; i < 100; i++) {
+      Account acc = new Account(
+        Name = 'Test Account ' + i,
+        Type = 'Prospect',
+        RecordTypeId = TaskTriggerHandler.recordTypeUS
+      );
+      accounts.add(acc);
+    }
+
+    insert accounts;
+  }
+
+  //inserting >200 tasks
+  @isTest
+  public static void insertNewTasks() {
+    List<Account> accounts = [
+      SELECT Id
+      FROM Account
+      LIMIT 42
+    ];
+
+    List<Task> tasks = new List<Task>();
+    for (Account acc : accounts) {
+      for (Integer i = 0; i < 5; i++) {
+        Task taskOther = new Task(
+          Subject = 'Other',
+          Type = 'Other',
+          WhatId = acc.id
+        );
+        tasks.add(taskOther);
```

Comment:
could be extracted into a helper method params: accounts, task type and maybe umber of tasks. helper method could do the insert as well